docs/core/compatibility/globalization/5.0/listseparator-value-change.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Breaking change: TextInfo.ListSeparator value change"
+description: Learn about the .NET 5.0 breaking change in where the default value of TextInfo.ListSeparator was changed between versions 5.0 and 5.0.1.
+ms.date: 12/10/2020
+---
+# TextInfo.ListSeparator values changed
+
+The default <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> values for different cultures have been reverted to their pre-.NET 5.0 values.
+
+## Change description
+
+In .NET 5.0.0, the default <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> values for different cultures changed. On Windows, the values changed as part of the [Globalization APIs use ICU libraries on Windows](icu-globalization-api.md) change. On Linux and macOS, instead of using decimal separators, the values for <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> were also obtained from International Components for Unicode (ICU).
+
+Starting in .NET 5.0.1, the values for <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> are identical to the values in .NET Framework and .NET Core 1.0 - 3.1. That is, on Windows, the <xref:System.Globalization.TextInfo.ListSeparator> values are obtained from National Language Support (NLS). On Linux and macOS, the <xref:System.Globalization.TextInfo.ListSeparator> values are obtained from the decimal separator values.
```

Comment:
> are identical to the values in .NET Framework and .NET Core 1.0 - 3.1

This is true only if running on Windows. .NET Core 1.0 - 3.1 on Linux/Mac will have same behavior as 5.0 when running on Winodws.